portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -44,7 +45,7 @@ public CommentsApiResponse(List<CommentsDatastore.Comment> comments) {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    Integer commentsLimit = Integer.parseInt(request.getParameter("commentsLimit"));
+    Integer commentsLimit = Integer.parseInt(request.getParameter(Constants.COMMENTS_LIMIT_PARAMETER));
```

Comment:
FYI if the parameter is not passed-in this will crash with NullPointerException (NPE)